en/deploy-on-gcp-gke.md

```diff
@@ -58,94 +56,97 @@ gcloud config set compute/region <gcp-region>
 
     ```shell
     gcloud container node-pools create pd --cluster tidb --machine-type n1-standard-4 --num-nodes=1 \
-      --node-labels=dedicated=pd --node-taints=dedicated=pd:NoSchedule
+        --node-labels=dedicated=pd --node-taints=dedicated=pd:NoSchedule
     gcloud container node-pools create tikv --cluster tidb --machine-type n1-highmem-8 --num-nodes=1 \
-      --node-labels=dedicated=tikv --node-taints=dedicated=tikv:NoSchedule
+        --node-labels=dedicated=tikv --node-taints=dedicated=tikv:NoSchedule
     gcloud container node-pools create tidb --cluster tidb --machine-type n1-standard-8 --num-nodes=1 \
-      --node-labels=dedicated=tidb --node-taints=dedicated=tidb:NoSchedule
+        --node-labels=dedicated=tidb --node-taints=dedicated=tidb:NoSchedule
     ```
 
-### Deploy TiDB Operator
+    The process might take a few minutes.
 
-To deploy TiDB Operator in the Kubernetes cluster, refer to the [*Deploy TiDB Operator* section](get-started.md#deploy-tidb-operator).
+## Deploy TiDB Operator
 
-### Deploy a TiDB cluster and the monitoring component
+To deploy TiDB Operator on GKE, refer to [deploy TiDB Operator](get-started.md#deploy-tidb-operator).
 
-1. Prepare the `TidbCluster` and `TidbMonitor` CR files:
+## Deploy a TiDB cluster and the monitoring component
 
-    {{< copyable "shell-regular" >}}
+This section describes how to deploy a TiDB cluster and its monitoring component on GCP GKE.
 
-    ```shell
-    curl -LO https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-cluster.yaml &&
-    curl -LO https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-monitor.yaml
-    ```
+### Create namespace
 
-2. Create `Namespace`:
+To create a namespace to deploy the TiDB cluster, run the following command:
 
-    {{< copyable "shell-regular" >}}
+{{< copyable "shell-regular" >}}
 
-    ```shell
-    kubectl create namespace tidb-cluster
-    ```
+```shell
+kubectl create namespace tidb-cluster
+```
 
-    > **Note:**
-    >
-    > A [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) is a virtual cluster backed by the same physical cluster. This document takes `tidb-cluster` as an example. If you want to use other namespace, modify the corresponding arguments of `-n` or `--namespace`.
+> **Note:**
+>
+> A [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) is a virtual cluster backed by the same physical cluster. This document takes `tidb-cluster` as an example. If you want to use other namespace, modify the corresponding arguments of `-n` or `--namespace`.
```

Comment:
this sentence reads weird, because `namespace` is not actually a "virtual cluster". maybe just refer to the kubernetes namespace?